src/core/evaluator.js

```diff
@@ -723,29 +723,70 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           return errorFont();
         }
       }
+      let fallbackFont = false;
+      let fallbackFontProperties = false;
       if (!fontRef) {
-        warn('fontRef not available');
-        return errorFont();
+        // warn('fontRef not available');
+        // return errorFont();
+
+        // fallback to some default font
```

Comment:
Nit: Please ensure that added comments always are complete sentences, i.e. starting with an upper-case letter and end with a dot.